src/OnboardingSPA/steps/SiteGen/Preview/stylesheet.scss

```diff
@@ -55,6 +55,8 @@
 		margin: 10px 10px 55px 10px;
 		flex-wrap: wrap;
 		max-width: 1440px;
+		align-items: center;
+		justify-content: center;
```

Comment:
![Screenshot 2024-01-31 at 7 15 14 PM](https://github.com/newfold-labs/wp-module-onboarding/assets/48691514/2f78b0e5-cfdf-4a25-a4a8-b7e2667c4dbf)
![Screenshot 2024-01-31 at 7 15 47 PM](https://github.com/newfold-labs/wp-module-onboarding/assets/48691514/93607437-aecf-41a4-a369-37de3a4ef22f)

Doesn't look right 👀

Reply:
@officiallygod  Have updated the code with by adding width to the parent container. The width will be different for devices.